pom.xml

```diff
@@ -30,6 +30,7 @@
         <module>load-balancer</module>
         <module>assembly</module>
         <module>phoenix-client</module>
+        <module>queryserver-orchestrator</module>
```

Comment:
If this is for phoenix-queryserver, why is this in the main phoenix repo. Could be that I just don't understand what the orchestrator is used for, but it seems like this should be in the new repo: https://github.com/apache/phoenix-queryserver

Reply:
Yes, it is under new repo phoenix-queryserver.

Reply:
Ok, issue is not with you. I'm just confused why there is a `phoenix-client` module for the queryserver repository. Sorry about that.